### PR TITLE
Add CloudBuild tests

### LINKED ISSUE #55: Add testing based on new approach

### PR DESCRIPTION
**Pull request by paulpalamarchuk**
* Fix #55

### CODE REVIEW

File: examples/https-gke/README.md [hunk 1]
```diff
@@ -9,162 +9,28 @@ This example creates an HTTPS load balancer to forward traffic to a custom URL m
 
 **Figure 1.** *diagram of Google Cloud resources*
 
-![architecture diagram](https://raw.githubusercontent.com/GoogleCloudPlatform/terraform-google-lb-http/master/examples/https-gke/diagram.png)
-
-## Change to the example directory
```
Comment (morgante) [inline examples/https-gke/README.md]: Don't remove the documentation, please leave these instructions.

File: examples/mig-nat-http-lb/README.md [hunk 2]
```diff
@@ -2,75 +2,28 @@
...
 [![button](http://gstatic.com/cloudssh/images/open-btn.png)](https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/GoogleCloudPlatform/terraform-google-lb-http&working_dir=examples/http-nat-gateway&page=shell&tutorial=README.md)
 
-<a href="https://concourse-tf.gcp.solutions/teams/main/pipelines/tf-examples-lb-http-nat-gw" target="_blank">
-<img src="https://concourse-tf.gcp.solutions/api/v1/teams/main/pipelines/tf-examples-lb-http-nat-gw/badge" /></a>
-
 This example creates a global HTTP forwarding rule to an instance group without external IPs. The instances access the internet via a NAT gateway.
 
 **Figure 1.** *diagram of Google Cloud resources*
 
-![architecture diagram](https://raw.githubusercontent.com/GoogleCloudPlatform/terraform-google-lb-http/master/examples/http-nat-gateway/diagram.png)
```
Comment (morgante) [inline examples/mig-nat-http-lb/README.md]: Please don't remove instructions.

File: examples/multi-backend-multi-mig-bucket-https-lb/README.md [hunk 3]
```diff
@@ -2,98 +2,35 @@
...
-<a href="https://concourse-tf.gcp.solutions/teams/main/pipelines/tf-examples-lb-https-content" target="_blank">
-<img src="https://concourse-tf.gcp.solutions/api/v1/teams/main/pipelines/tf-examples-lb-https-content/badge" /></a>
-
 This example creates an HTTPS load balancer to forward traffic to a custom URL map. The URL map sends traffic to the region closest to you with static assets being served from a Cloud Storage bucket. The TLS key and certificate is generated by Terraform using the [TLS provider](https://www.terraform.io/docs/providers/tls/index.html).
 
 **Figure 1.** *diagram of Google Cloud resources*
 
-![architecture diagram](https://raw.githubusercontent.com/GoogleCloudPlatform/terraform-google-lb-http/master/examples/multi-back-multi-mig-bucket-https-lb/diagram.png)
-
-## Change to the example directory
```
Comment (morgante) [inline examples/multi-backend-multi-mig-bucket-https-lb/README.md]: Please don't remove instructions.

File: examples/multi-mig-http-lb/README.md [hunk 4]
```diff
@@ -2,99 +2,28 @@
...
-<a href="https://concourse-tf.gcp.solutions/teams/main/pipelines/tf-examples-lb-http-basic" target="_blank">
-<img src="https://concourse-tf.gcp.solutions/api/v1/teams/main/pipelines/tf-examples-lb-http-basic/badge" /></a>
-
 This example creates a global HTTP forwarding rule to forward traffic to instance groups in the us-west1 and us-east1 regions.
 
 **Figure 1.** *diagram of Google Cloud resources*
 
-![architecture diagram](https://raw.githubusercontent.com/GoogleCloudPlatform/terraform-google-lb-http/master/examples/multi-mig-http-lb/diagram.png)
-
-## Change to the example directory
```
Comment (morgante) [inline examples/multi-mig-http-lb/README.md]: Please don't remove instructions.

File: examples/multiple-certs/README.md [hunk 5]
```diff
@@ -2,94 +2,33 @@
 
 [![button](http://gstatic.com/cloudssh/images/open-btn.png)](https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/GoogleCloudPlatform/terraform-google-lb-http&working_dir=examples/multiple-certs&page=shell&tutorial=README.md)
 
-<a href="https://concourse-tf.gcp.solutions/teams/main/pipelines/tf-examples-lb-https-multi-cert" target="_blank">
-<img src="https://concourse-tf.gcp.solutions/api/v1/teams/main/pipelines/tf-examples-lb-https-multi-cert/badge" /></a>
-
 This example shows how to use multiple certificates with the HTTPS Load Balancer module.
 
-## Change to the example directory
```
Comment (morgante) [inline examples/multiple-certs/README.md]: Please don't remove instructions.

File: examples/shared-vpc/README.md [hunk 6]
```diff
@@ -2,67 +2,17 @@
 
 This example shows how to use this module in a Shared VPC setup. In a service project, it creates a global HTTP forwarding rule to forward traffic to an instance group. Meanwhile it ensures the host project has firewall rules in place to allow the health check ingress.
 
-## Usage
```
Comment (morgante) [inline examples/shared-vpc/README.md]: Please don't remove instructions.

File: test/task_helper_functions.sh [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+  local pth="$1"
+  shift
+    find "${pth}" '(' \
+    -path '*/.git' -o \
+    -path '*/.terraform' -o \
+    -path '*/.kitchen' -o \
+    -path '*.svg' -o \
+    -path '*.png' -o \
+    -path './autogen' -o \
+    -path './test/fixtures/all_examples' -o \
```
Comment (morgante) [inline test/task_helper_functions.sh]: Why are we excluding these? Not clear why this repo needs a customized find_files.

File: examples/mig-nat-http-lb/README.md (hunk 2 above)
Comment (paulpalamarchuk) [reply examples/mig-nat-http-lb/README.md]: reverted

File: examples/multi-backend-multi-mig-bucket-https-lb/README.md (hunk 3 above)
Comment (paulpalamarchuk) [reply examples/multi-backend-multi-mig-bucket-https-lb/README.md]: reverted

File: examples/multi-mig-http-lb/README.md (hunk 4 above)
Comment (paulpalamarchuk) [reply examples/multi-mig-http-lb/README.md]: reverted

File: examples/multiple-certs/README.md (hunk 5 above)
Comment (paulpalamarchuk) [reply examples/multiple-certs/README.md]: reverted

File: examples/shared-vpc/README.md (hunk 6 above)
Comment (paulpalamarchuk) [reply examples/shared-vpc/README.md]: reverted

**Pull request merged**